pkgs/top-level/all-packages.nix

```diff
@@ -8340,7 +8340,7 @@ in
 
   vmtouch = callPackage ../tools/misc/vmtouch { };
 
-  vncdo = callPackage ../tools/admin/vncdo { };
+  vncdo = with python3Packages; toPythonApplication vncdo;
```

Comment:
```suggestion
  vncdo = python3Packages.toPythonApplication vncdo;
```

Reply:
This seems to trigger infinite recursion.

The only other type of invocation I see in `top-level/all-packages.nix` is e.g. 
```nix
vncdo = callPackage ../tools/admin/vncdo/default.nix { };
```
But that is now handled in `top-level/python-packages.nix`, so what I initially had might be the best syntax to use here.

Reply:
I see only two other packages that use the `python3Packages.toPythonApplication` syntax, so I think I'm going to stick with the former `with` syntax.